package.json

```diff
@@ -17,7 +18,9 @@
     "package": "cross-env NODE_ENV=production node -r babel-register -r babel-polyfill webpack/package.js",
     "package-all": "npm run package -- --all",
     "postinstall": "node node_modules/fbjs-scripts/node/check-dev-engines.js package.json",
+    "preinstall": "which electron || npm install electron@1.4.12",
```

Comment:
You specify it when specifying the whole package set here: https://github.com/input-output-hk/daedalus/pull/101/files#diff-5712e736e0de6ba170577f8472c398e9R1

We could add a note there to also bump electron in `pacakge.json`. What do you think? Sadly `package.json` doesn't allow comments.